packages/reports/src/_common/utility.js

```diff
@@ -35,18 +35,8 @@ const getStaticHash = () => {
     return static_hash;
 };
 
-class PromiseClass {
-    constructor() {
-        this.promise = new Promise((resolve, reject) => {
-            this.reject = reject;
-            this.resolve = resolve;
-        });
-    }
-}
-
 module.exports = {
     template,
     createElement,
     getStaticHash,
```

Comment:
@henry-deriv I believe `createElement` and `getStaticHash` are also dead code, We can remove them 🙇🏻